src/cascadia/TerminalApp/Jumplist.cpp

```diff
@@ -22,44 +23,33 @@ DEFINE_PROPERTYKEY(PKEY_AppUserModel_DestListLogoUri, 0x9F4C2855, 0x9F79, 0x4B39
 // - settings - The settings object to update the jumplist with.
 // Return Value:
 // - <none>
-void Jumplist::UpdateJumplist(std::shared_ptr<::TerminalApp::CascadiaSettings> settings)
+HRESULT Jumplist::UpdateJumplist(const TerminalApp::CascadiaSettings& settings)
 {

```

Comment:
this may need a function-level `try {} CATCH_RETURN;` to transform exceptions that come out of `winrt::com_ptr` into HRESULTs (same below)